src/components/Chat/MessageStage.tsx

```diff
@@ -60,7 +60,7 @@ export const MessageStage = ({ stage }: Props) => {
   }, [stage?.content, stage?.attachments?.length]);
 
   return (
-    <div className="block min-w-0 shrink rounded border border-gray-400 bg-gray-300 dark:border-gray-700 dark:bg-gray-900">
+    <div className="block min-w-0 shrink rounded border border-primary bg-layer-1">
```

Comment:
```suggestion
    <div className="block min-w-0 shrink rounded border border-secondary bg-layer-1">
```
![image](https://github.com/epam/ai-dial-chat/assets/20163971/2fbc3458-2d47-4e02-b1e6-1aafcd984050)